### PR TITLE
Reference guide - include relevant Weld version in the title

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

**Comment by weld-tester** [comment]
Triggering build using a merge of 12cc3043fc2c300ae8d01013e1684148491b338d on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 713 is now running using a merge of 12cc3043fc2c300ae8d01013e1684148491b338d on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/713

**Comment by weld-tester** [comment]
Build 713 outcome was SUCCESS using a merge of 12cc3043fc2c300ae8d01013e1684148491b338d on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/713
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/713

File: docs/reference/src/main/asciidoc/Weld_-_JSR-299_Reference_Implementation.asciidoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :numbered!:
 :icons: font
-= Weld - CDI Reference Implementation
+= Weld - CDI Reference Implementation ({weldVersion})
```
Comment (jharting) [inline docs/reference/src/main/asciidoc/Weld_-_JSR-299_Reference_Implementation.asciidoc]: I find this a bit confusing (a reader may confuse the version as CDI version). What do you think about the following scheme? Weld 2.2.11.Final - CDI Reference Implementation.

Comment (mkouba) [inline docs/reference/src/main/asciidoc/Weld_-_JSR-299_Reference_Implementation.asciidoc]: Yep, looks good...

**Comment by weld-tester** [comment]
Triggering build using a merge of b775df823f895c08650f8793159fa12ce35a3ddb on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 714 is now running using a merge of b775df823f895c08650f8793159fa12ce35a3ddb on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/714

**Comment by weld-tester** [comment]
Build 714 outcome was SUCCESS using a merge of b775df823f895c08650f8793159fa12ce35a3ddb on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/714
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/714

**Comment by jharting** [comment]
Merged, thanks!

**Pull request closed without merging**